pom.xml

```diff
@@ -239,6 +239,7 @@
             </goals>
             <configuration>
               <source>8</source>
+              <additionalOptions>-Xdoclint:none</additionalOptions>
```

Comment:
Why not use https://maven.apache.org/plugins/maven-javadoc-plugin/jar-mojo.html#failonerror and set that to false?

This seems to build and still print the warnings/errors
```
-              <additionalOptions>-Xdoclint:none</additionalOptions>
+              <failOnError>false</failOnError>
```

Reply:
applied as suggested